context-monitoring/src/main/java/de/atb/context/persistence/monitoring/MonitoringDataRepository.java

```diff
@@ -57,7 +50,7 @@
  * @author scholze
  * @version $LastChangedRevision: 143 $
  */
-public final class MonitoringDataRepository<Type extends IMonitoringDataModel<?, ?>> extends RepositorySDB<Type> implements
+public final class MonitoringDataRepository<Type extends IMonitoringDataModel<?, ?>> extends RepositoryTDB<Type> implements
```

Comment:
There are a few code duplications (e.g. lines 138-145 and 196-203, or 164-179 and 367-382) that could be cleaned up.

Reply:
should be better now